### PR TITLE
Add DR/SRS authoring skill

### PR DESCRIPTION
**Pull request by kin0992**
Add `dr-blacksmith`, a Markdown-first skill for creating and updating parent Design Review / Software Requirements Specification documents from PRDs, discovery, architecture, compliance, contracts, and delivery inputs.

Depends on #1980
Closes AEPDF-69

### CODE REVIEW

File: plugins/aiepfd/skills/dr-blacksmith/SKILL.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
+name: dr-blacksmith
+description: Create and update a structured parent Design Review / Software Requirements Specification (DR/SRS) from PRDs, discovery, architecture, compliance, contracts, and delivery inputs. Use whenever a user asks for a Design Review, DR, SRS, solution design, technical requirements document, architecture review, or to turn an accepted RFC into the operational design. Write Markdown first, preserve stable IDs, expose gaps without inventing decisions, maintain and verify a Use Case index without generating child Use Cases, invoke uc-engraver when detailed Use Cases are requested, and hand confirmed Confluence publication to the confluence-librarian skill.
```
Comment (gunzip) [inline plugins/aiepfd/skills/dr-blacksmith/SKILL.md]:
```suggestion
description: Create and update a structured Design Review / Software Requirements Specification (DR/SRS). Use whenever a user asks for a Design Review, DR, SRS, solution design, technical requirements document, architecture review, or to turn an accepted RFC into the operational design. Write Markdown first, preserve stable IDs, expose gaps without inventing decisions, invoke uc-engraver when detailed Use Cases are requested, and hand confirmed Confluence publication to the confluence-librarian skill.
```

File: plugins/aiepfd/skills/dr-blacksmith/SKILL.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+---
+
+# DR Blacksmith
+
+Forge the parent Design Review / Software Requirements Specification (DR/SRS)
```
Comment (gunzip) [inline plugins/aiepfd/skills/dr-blacksmith/SKILL.md]: thoughts: the concept of 'parent DR' does not appear anywhere before this statement. I'm unsure it's useful by itself.

File: plugins/aiepfd/skills/dr-blacksmith/SKILL.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
+---
+name: dr-blacksmith
+description: Create and update a structured parent Design Review / Software Requirements Specification (DR/SRS) from PRDs, discovery, architecture, compliance, contracts, and delivery inputs. Use whenever a user asks for a Design Review, DR, SRS, solution design, technical requirements document, architecture review, or to turn an accepted RFC into the operational design. Write Markdown first, preserve stable IDs, expose gaps without inventing decisions, maintain and verify a Use Case index without generating child Use Cases, invoke uc-engraver when detailed Use Cases are requested, and hand confirmed Confluence publication to the confluence-librarian skill.
+---
+
+# DR Blacksmith
```
Comment (gunzip) [inline plugins/aiepfd/skills/dr-blacksmith/SKILL.md:6]: suggestion: start stating what a DR is for and the expected target

File: plugins/aiepfd/skills/dr-blacksmith/SKILL.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+## When to use this skill
+
+Use this skill to:
+
+- create a DR/SRS from a PRD, discovery material, architecture notes, or a
+  task-centric intake;
+- update the current solution design after a material decision, accepted RFC,
+  new dependency, contract, risk, or Use Case;
+- prepare a Design Review document for stakeholder review or readiness;
+- keep a parent DR/SRS aligned with linked PRD, RFC, ADR, API, event, data,
```
Comment (gunzip) [inline plugins/aiepfd/skills/dr-blacksmith/SKILL.md]: same here: I'd get rid of the concept of 'parent DR'

File: plugins/aiepfd/skills/dr-blacksmith/SKILL.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+  questions, and decisions.
+- Keep detailed flows, edge cases, acceptance checks, and test notes in the
+  linked Use Case child documents. Do not recreate them in the parent.
+
+## Workflow
+
+1. **Collect source material.** Read the linked PRD and any available Service
+   Blueprint, Figma/design discovery, architecture diagrams, RFCs, ADRs,
+   OpenAPI/AsyncAPI specifications, Data Contracts, glossary, security and
+   privacy reviews, operational material, Jira board, and readiness evidence.
```
Comment (gunzip) [inline plugins/aiepfd/skills/dr-blacksmith/SKILL.md]:
```suggestion
1. **Collect source material.** Read the linked PRD and any other source provided by user or linked in the PDR```

File: plugins/aiepfd/skills/dr-blacksmith/SKILL.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+   trade-offs, non-functional requirements, compliance, technical contracts,
+   Use Case index, rollout and rollback, validation, monitoring, and readiness.
+4. **Clarify selectively.** Ask targeted questions for contradictions or
+   missing information that affects scope, behavior, ownership, architecture,
+   compliance, contracts, or readiness. Do not block a draft merely because
+   non-blocking information is unknown.
+5. **Write or update the draft.** Use the template and make gaps visible in
+   `Open questions, assumptions, and decisions`. Never convert a proposal into
+   a confirmed decision without evidence or user confirmation.
+6. **Maintain the Use Case index.** Add stable `UC-XX` entries with title,
```
Comment (gunzip) [inline plugins/aiepfd/skills/dr-blacksmith/SKILL.md:72]: thoughts: imho we don't need to explicitely link child pages as agents can retrieve them from the parent page

File: plugins/aiepfd/skills/dr-blacksmith/SKILL.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+   linked JTBD, priority, status, and child-page link when known. Flag missing
+   or stale child documents for the Use Case skill; do not write their bodies.
+   When child pages are published, require verification that the Confluence
+   parent page also contains the returned child URLs and no stale placeholders.
+7. **Validate the document.** Check that all stable sections are present,
```
Comment (gunzip) [inline plugins/aiepfd/skills/dr-blacksmith/SKILL.md]: suggestion: use a validation checklist (you may consider to have it as another resource / artifact)

File: plugins/aiepfd/skills/dr-blacksmith/SKILL.md [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+  never fill gaps with plausible technical detail.
+- Treat `accepted` RFCs as historical until their decision is visible in the
+  DR/SRS. An RFC that has not been propagated must not guide implementation.
+- Link existing contracts and reviews instead of copying their full contents.
+- Keep the parent document concise enough to govern the system; delegate
+  behavior-level detail to the Use Case skill.
+
+## Required parent sections
+
+The template follows the DR-01 source model:
```
Comment (gunzip) [inline plugins/aiepfd/skills/dr-blacksmith/SKILL.md]: thoughts: maybe we don't need to repeat the template outline here

File: plugins/aiepfd/skills/dr-blacksmith/SKILL.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+- Keep evidence traceable to its source artifact or stated input.
+- Distinguish confirmed facts, proposed design, assumptions, unresolved
+  questions, and decisions.
+- Keep detailed flows, edge cases, acceptance checks, and test notes in the
+  Use Case documents. Do not recreate them in the DR/SRS.
+
+## Workflow
+
+1. **Collect source material.** Read the linked PRD and any other source provided
+   by user or linked in the PDR.
```
[the remark this answers is not in the document]
Comment (kin0992) [reply plugins/aiepfd/skills/dr-blacksmith/SKILL.md]: Accepted. `PDR` is a typo; it should be `PRD`.

**Review by gunzip — APPROVED** [review]
